### ISSUE TITLE
Config file defaults not overridden by UI arguments

### ISSUE THREAD

**Issue by atte-wsp** [HSLdevcom/helmet-model-system#264]
If dev-config has `USE_FIXED_TRANSIT_COST` set to `true` and UI `false` the model run is performed with fixed cost. I think this is unwanted behavior and selections of UI should override the dev-config file defaults.

**Comment by zptro** [comment]
Yes, this is not very good. It seems that the script works as intended the other way around; UI `True` calls *do* override dev-config. This will need to be fixed both in `helmet.py` and the UI simultaneously.

We need to replace this line with `type=bool`:
https://github.com/HSLdevcom/helmet-model-system/blob/ccce73314fe56ec3e8eda6e9d22241b839da66d3/Scripts/helmet.py#L195

And we need to change this accordingly:
https://github.com/HSLdevcom/helmet-ui/blob/master/src/background/helmet_entrypoint_python_shell.js#L75

**Comment by johpiip** [comment]
This same problems applies to other flag parameters too. Here are both parser arguments and their equivalents in `dev-config.json`:
- `--run-agent-simulation` / `RUN_AGENT_SIMULATION`
- `--do-not-use-emme` / `!USE_EMME`
- `--del-strat-files` / `DELETE_STRATEGY_FILES`
- `--use-fixed-transit-cost` / `USE_FIXED_TRANSIT_COST`
- `--save-emme-matrices` / `SAVE_MATRICES_IN_EMME`

Some of these flags can be selected and deselected in the UI: right now, the only one is `--use-fixed-transit-cost`, but in the upcoming version also `--del-strat-files` and `--save-emme-matrices`. The CLI call is formulated so that only selected (`True`) flags are written in the call. This comes from `action="store_true"` definition. However, if the flag is missing, `helmet.py` takes the default value from `dev-config.json` even if we implicitly mean that the value of the flag is `False`. If the flag's value is `True` in `dev-config.json`, it will override the implicit falseness of the flag.

Some ways I can think to solve this:
- Writing the boolean explicitly like `python helmet.py --use-fixed-transit-cost False` as Jens suggested
    - I am not really fan of this as that kind of CLI calls are quite unusual at least from my point of view
- Ditching `dev-config.json` completely or at least all colliding parameters
- Do not read default values from `dev-config.json` unless explicitly told so
